lib/llm/src/model_card/model.rs

```diff
@@ -135,17 +138,21 @@ impl ModelDeploymentCard {
         ModelDeploymentCardBuilder::default()
     }
 
+    pub fn with_name_only(name: &str) -> ModelDeploymentCard {
```

Comment:
Can you summarize the name-only change here? Is it because the rest of the files or full MDC will be populated elsewhere with care_store changes? An inline comment on the motivation for it might help when jumping to the function definition from call sites - what the function does itself is clear, just not why it's used (at a glance)

Reply:
Good point, description added. Does this clarify?
```    
/// Create a ModelDeploymentCard where only the name is filled in.
///
/// Single-process setups don't need an MDC to communicate model details, but it
/// simplifies the code to assume we always have one. This is how you get one in those
/// cases. A quasi-null object: https://en.wikipedia.org/wiki/Null_object_pattern
```